engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/views/bulk-actions-environment.html

```diff
@@ -41,7 +41,7 @@ <h4 translate>Assign Environment and Content View</h4>
 
       <button class="btn btn-default fr"
               translate
-              ng-hide="denied('edit_content_hosts', contentHost)"
+              ng-hide="denied('edit_hosts', contentHost)"
```

Comment:
I know that I commented on the bulk page but I was thinking about the pages where there is a single content host.  I added another comment regarding my concern below.